editor/project_manager.cpp

```diff
@@ -900,7 +900,7 @@ class ProjectDialog : public ConfirmationDialog {
 		rs_button->set_meta("driver_name", "GLES2");
 		rvb->add_child(rs_button);
 		l = memnew(Label);
-		l->set_text(TTR("- Lower visual quality\n- Some features not available\n- Works on most hardware\n- Recommended for web games"));
+		l->set_text(TTR("- Lower visual quality\n- Obsolete and less accurate API, which produces not so fast code\n- Some features not available\n- Works on most hardware\n- Recommended for web games"));
```

Comment:
I don't think this is a useful addition. If it's "obsolete, less accurate, not fast", why would we even propose it to users?

Reply:
Okay, you have reason. But I just wanted to emphasize one more notable difference between them. How I should write it correctly? Or maybe delete at all, if not needed.

Reply:
I think you can keep the current description and replace "All features available" with "Some features not implemented yet".

Reply:
Better now.